test/metamorpho_tests.tree

```diff
@@ -237,13 +237,9 @@
         │   └── it should emit ReallocateSupply(id, suppliedAssets, suppliedShares)
         ├── when totalWithdrawn > totalSupplied
         │   └── it should add totalWithdrawn - totalSupplied to idle
-        ├── when totalWithdrawn <= totalSupplied
-        │   └── it should compute idleSupplied = totalSupplied - totalWithdrawn
-        │       ├── when idle < idleSupplied
-        │       │   └── revert with InsufficientIdle()
-        │       └── when idle >= idleSupplied
-        │           └── it should remove idleSupplied from idle
-        └── it should emit ReallocateIdle(idle)
+        └── when totalWithdrawn != totalSupplied
+            └── revert with InconsistentReallocation()
```

Comment:
We can remove the two lines above and there are many other occurrences of `idle` in this file. This is as good a time as any to get rid of this file :)